source/ios/AdaptiveCards/ADCIOSVisualizer/ADCIOSVisualizerTests/ADCIOSVisualizerTests.mm

```diff
@@ -455,6 +456,11 @@ - (void)testSharedEnumsCompatabilityTest
     XCTAssertTrue(static_cast<int>(AdaptiveCards::CardElementType::ToggleInput) == ACRToggleInput);
     XCTAssertTrue(static_cast<int>(AdaptiveCards::CardElementType::Unknown) == ACRUnknown);
     XCTAssertTrue(static_cast<int>(AdaptiveCards::CardElementType::RichTextBlock) == ACRRichTextBlock);
+    XCTAssertTrue(static_cast<int>(AdaptiveCards::ActionType::ShowCard) == ACRShowCard);
```

Comment:
>XCTAssertTrue(static_cast<int>(AdaptiveCards::ActionType::ShowCard) == ACRShowCard); [](start = 4, length = 84)

Since these are static values, it isn't really necessary to have a runtime test to validate these things. You can do static_assert, or something equivalent, and do the test at compile time.

Reply:
thanks; 

---
In reply to: [338166619](https://github.com/microsoft/AdaptiveCards/pull/3526#discussion_r338166619) [](ancestors = 338166619)